<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Otter::WebsiteInformationDialog</class>
 <widget class="QDialog" name="Otter::WebsiteInformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>338</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="hostLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="stateLayout">
     <item>
      <widget class="QLabel" name="stateIconLabel">
       <property name="minimumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stateLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="connectionTab">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="certificateHeaderLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Certificate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="certificateNameLabel">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="certificateExpirationDateLabel">
         <property name="text">
          <string>Expiration date:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="certificateEffectiveDateLabel">
         <property name="text">
          <string>Effective date:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="cipherHeaderLabel">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cipher</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="cipherProtcolLabel">
         <property name="text">
          <string>Protocol:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="cipherAuthenticationMethodLabel">
         <property name="text">
          <string>Authentication method:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="cipherKeyExchangeMethodLabel">
         <property name="text">
          <string>Key exchange method:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="cipherEncryptionMethodLabel">
         <property name="text">
          <string>Encryption method:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="certificateLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="certificateDetailsButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Detailsâ€¦</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="Otter::TextLabelWidget" name="certificateNameLabelWidget" native="true"/>
       </item>
       <item row="2" column="1">
        <widget class="Otter::TextLabelWidget" name="certificateEffectiveDateLabelWidget" native="true"/>
       </item>
       <item row="3" column="1">
        <widget class="Otter::TextLabelWidget" name="certificateExpirationDateLabelWidget" native="true"/>
       </item>
       <item row="5" column="1">
        <widget class="Otter::TextLabelWidget" name="cipherProtcolLabelWidget" native="true"/>
       </item>
       <item row="6" column="1">
        <widget class="Otter::TextLabelWidget" name="cipherAuthenticationMethodLabelWidget" native="true"/>
       </item>
       <item row="7" column="1">
        <widget class="Otter::TextLabelWidget" name="cipherEncryptionMethodLabelWidget" native="true"/>
       </item>
       <item row="8" column="1">
        <widget class="Otter::TextLabelWidget" name="cipherKeyExchangeMethodLabelWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="permissionsTab">
      <attribute name="title">
       <string>Permissions</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Otter::TextLabelWidget</class>
   <extends>QWidget</extends>
   <header>src/ui/TextLabelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
